pepcli/test/test.go

```diff
@@ -95,3 +102,12 @@ func dump(r pdp.Response, f io.Writer) error {
 	_, err := fmt.Fprintf(f, "%s\n", strings.Join(lines, "\n"))
 	return err
 }
+
+func sortDelimitedListOfStrings(unsortedCommaList string, delimiter string) string {
+	list := strings.Split(unsortedCommaList, delimiter)
```

Comment:
The list is JSON formatted string list with no brackets around (as I can recall). To split it correctly you can add pair of "[" "]" and unmarshal it with json package to []string.